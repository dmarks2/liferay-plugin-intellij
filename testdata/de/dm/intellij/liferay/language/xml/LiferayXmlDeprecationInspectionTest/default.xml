<?xml version="1.0" encoding="UTF-8"?>
<warning descr="The descriptor XML DTD versions should be matched with the Liferay version"><!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.0.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_0_0.dtd"></warning>
<resource-action-mapping>
</resource-action-mapping>