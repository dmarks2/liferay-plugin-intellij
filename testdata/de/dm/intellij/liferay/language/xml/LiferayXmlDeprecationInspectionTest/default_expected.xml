<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.4.0//EN"
        "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4_0.dtd">
<resource-action-mapping>
</resource-action-mapping>